cpp/mcap/include/mcap/visibility.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#if defined _WIN32 || defined __CYGWIN__
```

Comment:
can we get some indentation in this file so it is legible?

Reply:
yes :+1: i think some windows vscode setting auto-changed this, it started indented - will fix

Reply:
oh it was also clang-format didn't like it, I added the setting to allow for preprocessor indents

Reply:
I've also seen some files keep the `#` at the beginning of the line and put the spaces right after it. Not sure which is better 🤷

Reply:
oh I see that's exactly what you did 👍